.github/ISSUE_TEMPLATE/04_user_support.yml

```diff
@@ -0,0 +1,27 @@
+name: User Support
+description: Do you need assistance with admiral?
+title: "Guidance and support on how to use admiral"
+labels: ["user support"]
+assignees:
+  - octocat
+body:
```

Comment:
Should we mention that for bugs, enhancements, and documentation updates the corresponding issue types should be used?